SPECS/ltp/ltp.spec

```diff
@@ -76,8 +74,6 @@ make autotools
 %make_install
 
 %check
-# Disabling cloning of git submodules - already provided in Source1
-sed -i "s/git submodule.*/@echo 'Skipping submodule init - already provided.'/" tools/sparse/Makefile
```

Comment:
Are the ptests still passing with this line removed?

Reply:
yes